firestore-emulator/typescript-quickstart/package.json

```diff
@@ -3,22 +3,19 @@
   "version": "1.0.1",
   "description": "Cloud Firestore emulator testing, with TypeScript",
   "scripts": {
-    "pretest": "eslint test/*.ts && tsc",
+    "format": "prettier --write **/*.ts",
+    "pretest": "tsc",
     "test": "mocha"
   },
   "devDependencies": {
-    "@firebase/testing": "0.3.0",
+    "@firebase/app": "^0.3.5",
+    "@firebase/testing": "0.3.3",
     "@types/mocha": "5.2.5",
-    "eslint": "5.7.0",
-    "eslint-config-google": "0.11.0",
     "mocha": "5.2.0",
     "mocha-typescript": "1.1.17",
+    "prettier": "^1.15.2",
     "source-map-support": "0.5.9",
     "ts-node": "7.0.1",
-    "typescript": "3.1.3",
-    "typescript-eslint-parser": "20.0.0"
-  },
-  "dependencies": {
-    "firebase": "^5.5.5"
+    "typescript": "3.1.3"
```

Comment:
Shouldn't typescript be a devDependency?  Any reason it got moved out?

Reply:
This is a confusing-looking diff, it's still a dev dependency.